source/models/options.rst

```diff
@@ -38,11 +40,13 @@ desired_processing     n/a              *CURRENTLY UNSUPPORTED* list of speech-r
     {
       "media_url": "https://support.rev.com/hc/en-us/article_attachments/200043975/FTC_Sample_1_-_Single.mp3",
       "metadata": "sample metadata",
-      "callback_url": "https://www.example.com/callback"
+      "callback_url": "https://www.example.com/callback",
+      "skip_diarization": "false"
```

Comment:
I think we need to include in `/GET transcript` endpoint a sample of when `skip_diarization` is `true`

Reply:
<img width="759" alt="screen shot 2018-10-24 at 10 51 45 am" src="https://user-images.githubusercontent.com/5668128/47450807-033bcc80-d77b-11e8-93bd-fadecaa478b6.png">
<img width="775" alt="screen shot 2018-10-24 at 10 51 35 am" src="https://user-images.githubusercontent.com/5668128/47450808-033bcc80-d77b-11e8-958b-dd42efefb659.png">

Reply:
What about JSON format?

Reply:
Added to the Transcript Model page